tests/ui/mir/inline-wrong-abi.rs

```diff
@@ -1,4 +1,4 @@
-//@ compile-flags: -Zpolymorphize=on -Zinline-mir=yes -Zmir-opt-level=0
```

Comment:
Can u change this to `-Zvalidate-mir`? The presence of `-Zpolymorphize` this test was just to observe (and thus compute and validate) the optimized MIR before codegen.

Reply:
I really wasn't sure what of the affected tests should be salvaged. Thanks for the guidance ❤️